src/scanner.ll

```diff
@@ -159,23 +161,23 @@
                                       }
 0b[0-1][0-1]*                         {
                                         try {
-                                          return yy::parser::make_NUMBER((RamDomain)std::stoull(yytext+2, NULL, 2), yylloc);
+                                          return yy::parser::make_NUMBER(souffle::stord(yytext+2, NULL, 2), yylloc);
```

Comment:
While you're changing this, NULL -> nullptr

Reply:
Done.